r_markdown/emissions.Rmd

```diff
@@ -144,32 +126,174 @@ output:
+                                       dplyr::rename(mass_on = on, mass_off = off))
+```
+
+* extract temperature for periods of interest
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  e_temp <-  dplyr::select(temp, id, datetime, time, tamb, thood)
+
+  e_temp <- filter_temp(e_times, e_temp)
+```
+
+* temperature timeseries
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE, fig.width=12, fig.height=80}
+  ggplot(e_temp, aes(datetime, tamb, group = rep, colour = rep)) +
+    geom_point() +
+    facet_wrap(~id, scales = "free", ncol = 3) +
+    theme_minimal()
+```
+
+* temperature change
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  e_temp <- dplyr::group_by(e_temp, id, rep) %>%
+            dplyr::summarise(t_min = quantile(tamb, 0.05, na.rm = TRUE),
+                             t_max = quantile(tamb, 0.95, na.rm = TRUE))
+```
+
+* merge data
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  e_data <- dplyr::left_join(e_times, e_pot_water_mass, by = c("id", "rep")) %>%
+            dplyr::left_join(e_temp, by = c("id", "rep")) %>%
+            dplyr::left_join(e_pot_mass, by = "id") %>%
+            dplyr::left_join(e_pot, by = c("id", "rep")) %>%
+            dplyr::mutate(mass_pot = ifelse(pot == "a", a, NA),
+                          mass_pot = ifelse(pot == "b", b, mass_pot),
+                          id = as.factor(id)) %>%
+            dplyr::select(-a, -b)
+```
 
-## ecoc
+* energy applied
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  e_data <- dplyr::mutate(e_data,
+                          mass_evap = mass_on - mass_off,
+                          mass_h2o = mass_on - mass_pot,
+                          dt = t_max - t_min,
+                          j_heat = mass_h2o * 4.186 * dt, # g * J/(g.oC) * oC
+                          j_vap = mass_evap * 2260,
+                          joules = j_heat + j_vap)
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  kable(dplyr::slice(e_data, 1:5),
+        align = 'c', caption = "Energy Number")
+```
+
+* energy applied per test
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  energy <- dplyr::select(e_data, id, joules) %>%
+            dplyr::group_by(id) %>%
+            dplyr::summarise(joules = sum(joules, na.rm = TRUE))
+```
+
+```{r, echo=FALSE, warning=FALSE, message=FALSE, results='markup', cache=FALSE, fig.width=12, fig.height=8}
+  p_data <- dplyr::left_join(dplyr::filter(samples, type == "SF"),
+                             energy,
+                             by = "id")
+
+  ggplot(p_data, aes(id, joules, colour = stove)) +
+  geom_point() +
+  theme_minimal() +
+  theme(axis.text.x = element_text(angle = 60, hjust = 1)) +
+  theme(legend.position = "top")
+  ylab("energy applied to stove (J)")
+  
```

Comment:
I can take a look at these values to see if they make sense after we get more the the temperature oddities sorted out?